src/pages/careers/locations/asuncion/index.tsx

```diff
@@ -9,7 +9,14 @@ import { localize, WithIntl } from 'components/localization'
 const query = graphql`
     query {
         asuncion: file(relativePath: { eq: "careers/asuncion.png" }) {
-            ...heroImage
+            childImageSharp {
```

Comment:
can you please create an `image-fragments` in `src/components/graphql/image-fragments.ts` and reuse it other files as well? thanks :pray:

Reply:
Ok